AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DirectionalLightAdded.py

```diff
@@ -74,13 +224,40 @@ def AtomEditorComponents_DirectionalLight_AddedToEntity():
     5) Enter/Exit game mode.
     6) Test IsHidden.
     7) Test IsVisible.
-    8) Add Camera entity.
-    9) Add Camera component to Camera entity
-    10) Set the Directional Light component property Shadow|Camera to the Camera entity.
-    11) Delete Directional Light entity.
-    12) UNDO deletion.
-    13) REDO deletion.
-    14) Look for errors and asserts.
+    8) Set Color to purple, then back to default.
+    9) Intensity mode set to Nit then back to Ev100.
+    10) Set Intensity value to 10 then back to default of 4.
+    11) Set Angular Diameter to 1.0 then back to default of 0.5.
+    12) Add Camera entity.
+    13) Add Camera component to Camera entity.
+    14) Set the Directional Light component property Shadow|Camera to the Camera entity.
+    15) Shadow Far Clip set to 2000 then back to the default of 100.
+    16) Shadow Map size set to 256 then back to the default of 1024.
+    17) Cascade Count set to 2 then back to the default of 4.
+    18) Split ratio can be set to 0.3 then back to the default of 0.9.
+    19a) Turn Automatic Splitting off for subsequent tests.
+    20) Far depth cascade can be set to test value then back to the default value.
+    19b) Turn Automatic Splitting back on.
+    21a) Turn Cascade correction on for subsequent test.
+    22) Ground Height can be set to 10 then back to the default of 0.0.
+    21b) Turn Cascade correction back off.
+    23) Turn Debug Coloring on then back off.
+    24) Set Shadow filter method to PCF for subsequent tests.
+    25) Filtering sample count size set to 50 then back to the default of 32.
+    26) Turn Shadow Receiver Plane Bias Enable off then back on.
+    27) Shadow Bias can be set to 0.1 then back to the default of 0.002.
+    28) Normal Shadow Bias can be set to 7.0 then back to the default of 2.5.
+    29) Turn Blend Between Cascades on then back off.
+    30) Set Shadow filter method to ESM, PCF+ESM, then back to default of None.
+    31) Turn Fullscreen Blur off then back on.
+    32) Fullscreen Blur Strength can be set to 7.0 then back to the default of 2.5.
+    33) Fullscreen Blur Sharpness can be set to 240.0 then back to the default of 2.5.
```

Comment:
The `Tests.fullscreen_blur_sharpness_set_to_default` tuple has a string that mentions a default value of 50, but this docstring says the default is 2.5 - also down in the test code comment this needs updating as well on line 711

Reply:
fixed